packages/settings-library/src/settings_library/email.py

```diff
@@ -8,6 +9,12 @@
 from .basic_types import PortInt
 
 
+class EmailProtocol(Enum):
```

Comment:
```suggestion
class EmailProtocol(str, Enum):
```
Since we are using Pydantic, this help with decoding when serialising. SEE [string][https://pydantic-docs.helpmanual.io/usage/types/] for more